docs/reference/module-options.md

```diff
@@ -811,3 +843,198 @@ module.exports = {
 }
 ```
 
+
+## Options for widget modules
+
+Option settings in this section apply to all widgets (modules that extend `@apostrophecms/widget-type`).
+
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`className`](#classname) | String | A project-level class to apply to core widget templates. |
+| [`icon`](#icon) | String | Select an available icon to include with the label in area menus. |
+| [`label`](#label-for-widgets) | String | The readable label for the widget type. |
+
+<!-- | [`scene`](#scene) | null | description | -->
+<!-- | [`contextual`](#contextual) | Boolean | description | -->
+
+### `className`
+
+Official Apostrophe widget templates support adding an html class from the `className` module option. The class is applied to the outer, wrapping HTML element in the widget template. This helps developers style the widget in project-level CSS using their namespacing and without targeting Apostrophe data attributes.
+
+This option can be supported in custom, project-level widget templates (referenced with `data.manager.options.className` in template), but it may not be necessary. The option does not automatically have any effect in custom widget templates without use of `data.manager.options.className`.
```

Comment:
I'm confused by this statement. Help me out?

Reply:
it's confusing.

Reply:
i'll delete it. not helping much.